### PR TITLE
Regenerating Service Bus administration client

### PR DESCRIPTION
**Pull request by conniey**
# Description

* Regenerates service bus administration client
* Implementation classes/models have \*Impl automatically added to them to avoid name conflicts with public API
* Adds Autorest customizations for changes that were manually made before.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/servicebus/azure-messaging-servicebus/swagger/src/main/java/AdministrationClientCustomization.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import com.github.javaparser.ast.expr.StringLiteralExpr;
+import org.slf4j.Logger;
+
+import java.util.HashSet;
+import java.util.Objects;
+
+/**
+ * Customization class for Administration client for Service Bus
+ */
+public class AdministrationClientCustomization extends Customization {
```
Comment (anuchandy) [inline sdk/servicebus/azure-messaging-servicebus/swagger/src/main/java/AdministrationClientCustomization.java:22]: thanks for the Customization type, it saves so much time (also nice to see detailed comments on this class.)

**Review by anuchandy — APPROVED** [review]
thank you so much!

File: sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescriptionImpl.java [hunk 2]
```diff
@@ -121,29 +115,26 @@ public final class QueueDescription {
...
 
     private static final class AuthorizationRulesWrapper {
-        @JacksonXmlProperty(localName = "AuthorizationRule",
-            namespace = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect")
-        private final List<AuthorizationRuleImpl> items;
 
-        @JsonCreator
-        private AuthorizationRulesWrapper() {
-            this.items = Collections.emptyList();
-        }
```
Comment (anuchandy) [inline sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescriptionImpl.java:140]: Looks like the default Ctr patch introduced in the pr https://github.com/Azure/azure-sdk-for-java/pull/29508 got overwritten by the autorest. :(

Comment (conniey) [reply sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/models/QueueDescriptionImpl.java:140]: Urgh. Let me add that into the customisations as well.

**Pull request merged**